commons/all/src/main/java/org/infinispan/commons/util/IntSet.java

```diff
@@ -220,4 +220,16 @@ default int[] toIntArray() {
     * @return the index of the next set bit, or -1 if there is no such bit
     */
    int nextSetBit(int fromIndex);
+
+   /**
+    * Get the last bit set.
+    *
+    * @return The last bit set, or -1 if no bit is set.
+    */
+   default int lastSetBit() {
```

Comment:
We should implement this method for our various IntSet impls.

Reply:
Ended up not needed the method :smiley: